package.json

```diff
@@ -45,7 +45,7 @@
     "prebuild:plugins:prod": "npm run install:plugins",
     "build:plugins:prod": "NODE_ENV=production npm run build:plugins",
     "postbuild:plugins:prod": "npm --prefix plugins prune --production",
-    "prebuild:frontend": "npm --prefix frontend install",
+    "prebuild:frontend": "npm --prefix frontend install --production=false",
```

Comment:
Hey @arpitnath, was wondering why we need this change? It seems to have introduced a lot of package changes.

Reply:
@gondar00

Reply:
This change is related to Heroku build size being more than 500MB after adding a few packages, which I have explained in detail in this issue [#3797](https://github.com/ToolJet/ToolJet/issues/3797)